src/oas.md

```diff
@@ -4805,8 +4805,9 @@ Certain fields allow the use of Markdown which can contain HTML including script
 
 | Version | Date | Notes |
 | ---- | ---- | ---- |
-| 3.2.0 | TBD | Release of the OpenAPI Specification 3.2.0 |
-| 3.1.2 | TBD | Patch release of the OpenAPI Specification 3.1.2 |
+| 3.2.1 | TBD | Release of the OpenAPI Specification 3.2.1 |
+| 3.2.0 | 2025-09-19 | Release of the OpenAPI Specification 3.2.0 |
+| 3.1.2 | 2025-09-19 | Patch release of the OpenAPI Specification 3.1.2 |
```

Comment:
These date updates should also (IMHO) be updated in the live 3.2.0 release spec, and in the v3.3-dev branch as well.